gammapy/spectrum/spectrum_analysis.py

```diff
@@ -1,12 +1,18 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import absolute_import, division, print_function, unicode_literals
 import logging
+from astropy import wcs
+
 import numpy as np
 from astropy.coordinates import Angle, SkyCoord
 from astropy.extern import six
-from ..image import ExclusionMask
+from astropy.wcs.utils import skycoord_to_pixel
+
+from gammapy.extern import pathlib
```

Comment:
@cdeil : Do you know how to teach PyCharm to make import relative when optimizing imports?